src/modules/billing/services/batch-service.js

```diff
@@ -46,6 +48,22 @@ const getBatches = async (page = 1, perPage = Number.MAX_SAFE_INTEGER) => {
   };
 };
 
+const deleteBatch = async batchId => {
+  // TODO: This function is a stub and will eventually
+  // have a defined signature and response behaviour.
+  // If this fails, the batch should not be deleted from
+  // the WRLS database tables.
+  await chargeModuleBatchConnector.deleteBatch();
+
+  await repos.billingBatchChargeVersionYears.deleteByBatchId(batchId);
```

Comment:
Is deleting from each table preferable to setting up 'on delete cascade' and then just deleting the batch?

Reply:
I prefer it because it is explicit and testable.